gradle/libs.versions.toml

```diff
@@ -53,6 +54,7 @@ room-runtime = { module = "androidx.room:room-runtime", version.ref = "room" }
 room-compiler = { module = "androidx.room:room-compiler", version.ref = "room" }
 room-ktx = { module = "androidx.room:room-ktx", version.ref = "room" }
 date-time-picker = { module = "network.chaintech:kmp-date-time-picker", version.ref = "dateTimePicker" }
+androidx-ui-android = { group = "androidx.compose.ui", name = "ui-android", version.ref = "uiAndroid" }
```

Comment:
I don't see this dependency used anywhere. Can you remove this along with the version definition `uiAndroid`